config/packages/prod/monolog.yaml

```diff
@@ -5,6 +5,7 @@ monolog:
             action_level: error
             handler: nested
             excluded_http_codes: [404, 405]
+            buffer_size: 50 # How many messages should be saved? Prevent memory leaks
```

Comment:
Isn't this way, way way too few? maybe 50,000 messages might be more reasonable. Is this really a memory leak issue or a drive space thing with runaway logs?

Reply:
This is how it's set by Symfony, not my doing.

Reply:
So,  buffer size is just that, it's not a limit of messages. It will write keep 50 messages in it's buffer, then write. It is not a limitation of how many messages.